src/renderers/dom/shared/__tests__/ReactDOMComponent-test.js

```diff
@@ -1134,6 +1134,29 @@ describe('ReactDOMComponent', () => {
         'onLoad called'
       );
     });
+
+      var container = document.createElement('div');
+      var loadEvent = document.createEvent('Event');
+      var errorEvent = document.createEvent('Event');
+
+      loadEvent.initEvent('load', false, false);
```

Comment:
I believe `initEvent` is deprecated, would you consider using `Event()` and `dispatchEvent` instead?